mlrun/model_monitoring/db/stores/base/store.py

```diff
@@ -19,6 +19,7 @@
 
 
 class StoreBase(ABC):
+    type: str = ""
```

Comment:
I think that's more accurate + you don't need a default value:
```suggestion
    type: typing.ClassVar[str]
```

Reply:
nice, I've updated `TSDBConnector` as well